src/org/traccar/protocol/OsmAndProtocolDecoder.java

```diff
@@ -38,8 +40,11 @@
 
 public class OsmAndProtocolDecoder extends BaseProtocolDecoder {
 
+    private String speedUnits;
+
     public OsmAndProtocolDecoder(OsmAndProtocol protocol) {
         super(protocol);
+        speedUnits = Context.getConfig().getString("osmand.speedUnits", "kn");
```

Comment:
Please make it consistent with other places:

https://github.com/tananaev/traccar/blob/master/src/org/traccar/protocol/Tk103ProtocolDecoder.java#L256

Possibly updating TK103 decoder would be even better.

Reply:
Don't you mind if I move common code to `BaseProtocolDecoder` ?

Reply:
I guess it makes sense.